hack/testing/run_daemon_cri_integration.sh

```diff
@@ -147,7 +157,15 @@ main() {
     cat "${pouchd_log}"
     exit ${code}
   fi
-  integration::run_daemon_cri_test_cases
+  integration::run_daemon_cri_test_cases "${cri_runtime}"
+}
+
+main() {
+    integration::install_cni
+    cri_runtime="v1alpha1"
```

Comment:
please run `integration::run_cri_test "${cri_runtime}"` don't use global `cri_runtime`. If you have to, please use `local` to consume the `cri_runtime` to avoid the global variable pollution